cotton/salt/pillar.py

```diff
@@ -41,12 +41,12 @@ def get_unrendered_pillar_locations(include_project=True):
 
 
 def _get_projects_location():
-    fab_location = os.path.dirname(env.real_fabfile)
-    return os.path.abspath(os.path.join(fab_location, '../config/projects/'))
```

Comment:
Is the path '../config/projects' still relevant as we are moving to a set of pillar dirs which is made up of

- local to project deploy repo
- shared repo with common data

The jenkins job which runs the deployment scenarios would typically checkout the shared common pillar repo and the deploy repo and then use paths within them for the pillar data.